CHANGELOG.md

```diff
@@ -3,6 +3,160 @@
+
+- [DataGridPro] Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+
+  ```diff
+  - import { XGrid } from '@mui/x-data-grid-pro';
+  + import { DataGridPro } from '@mui/x-data-grid-pro';
+  ```
+
+- [core] Move packages to `@mui` scope (#2341) @oliviertassinari
+
+  ```diff
+  - import { DataGrid } from '@material-ui/data-grid';
+  + import { DataGrid } from '@mui/x-data-grid';
+  ```
+
+  ```diff
+  - import { LicenseInfo } from '@material-ui/x-grid';
+  + import { LicenseInfo } from '@mui/x-data-grid-pro';
+  ```
+
+- [DataGrid] Remove `apiRef` from all `options` callback params (#2312) @DanailH
+
+  ```diff
+  export interface GridColumnHeaderParams {
+    * The column of the current header component.
+    */
+    colDef: GridStateColDef;
+    -  /**
+    -  * API ref that let you manipulate the grid.
+    -  */
+    - api: any;
+  }
+  ```
+
+  ```diff
+  export interface GridColumnOrderChangeParams {
+    * The old column index.
+    */
+    oldIndex: number;
+    -  /**
+    -   * API ref that let you manipulate the grid.
+    -   */
+    -  api: any;
+  }
+  ```
+
+  ```diff
+  export interface GridColumnResizeParams {
+    * The column of the current header component.
+    */
+    colDef: GridStateColDef;
+  -  /**
+  -   * API ref that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  ```diff
+  export interface GridColumnVisibilityChangeParams {
+    * The column of the current header component.
+    */
+    colDef: GridStateColDef;
+  -  /**
+  -   * API ref that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  ```diff
+  export interface GridRowParams {
+    * All grid columns.
+    */
+    columns: GridColumns;
+  -  /**
+  -   * GridApiRef that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  ```diff
+  export interface GridRowScrollEndParams {
+    * The grid visible columns.
+    */
+    visibleColumns: GridColumns;
+  -  /**
+  -   * API ref that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  - All DataGrid props callbacks don't provide access to `api` anymore. To access `api` you need to use `DataGridPro` and access it from the details parameter.
+  ```
+  <DataGridPro
+    onColumnResize={(params, event, details) => console.log(details.api)}
+  />
+  ```
+
+  - `GridStateChangeParams` is removed, instead `onStateChange` first param is `GridState`.
+  ```diff
+  - onStateChange?: (params: GridStateChangeParams, event: MuiEvent<{}>, details: any) => void;
+  + onStateChange?: (state: GridState, event: MuiEvent<{}>, details: any) => void;
+  ```
+
+#### Changes
+
+- [DataGrid] Don't apply `box-sizing: border-box` on everything (#2330) @m4theushw
+- [DataGrid] Fix keyboard navigation header regression (#2342) @oliviertassinari
+- [DataGrid] Keep prop-types in the same file (#2345) @oliviertassinari
+- [DataGrid] Remove `apiRef` from all `options` callback params (#2312) @DanailH
+- [DataGrid] Remove private API from the export (#2299) @oliviertassinari
```

Comment:
A breaking change?

Reply:
In the PR it's said not to list that breaking change.